pkg/binding/nested/field_copy.go

```diff
@@ -0,0 +1,23 @@
+package nested
+
+import (
+	"bytes"
+	"strings"
+
+	"k8s.io/client-go/util/jsonpath"
+)
+
+func GetNested(obj map[string]interface{}, path_segments ...string) (string, bool, error) {
```

Comment:
Let's rename this function to reflect better what it does: how about `getValuesByJSONPath`?